package.json

```diff
@@ -55,7 +55,8 @@
     "test-ci": "taper tests/test-*.js",
     "test-cov": "istanbul cover tape tests/test-*.js",
     "test-browser": "node tests/browser/start.js",
-    "lint": "standard"
+    "lint": "standard",
+    "doctoc": "doctoc --title='## Table of contents' --github README.md"
```

Comment:
Could be changed to `Table of Contents`

Reply:
Updated, thanks.

Reply:
Just a reminder, `contents` is currently being used in the README.md